src/realm-settings/PoliciesTab.tsx

```diff
@@ -43,9 +45,14 @@ export const PoliciesTab = () => {
   const [code, setCode] = useState<string>();
   const [tablePolicies, setTablePolicies] =
     useState<ClientPolicyRepresentation[]>();
-
   const refresh = () => setKey(key + 1);
 
+  type EnabledConfigPolicyForm = {
+    [index: string]: boolean;
+  };
+
+  const form = useForm<EnabledConfigPolicyForm>({ mode: "onChange" });
```

Comment:
TypeScript has a useful type for these kind of objects called [`Record`](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type) which you could use here. Both options are equally valid though, so it comes down to personal preference.

```suggestion
  const form = useForm<Record<string, boolean>>({ mode: "onChange" });
```

Reply:
Ah yes, very cool! Thank you for suggesting this